org.tryton.Tryton-70.yml

```diff
@@ -0,0 +1,37 @@
+id: org.tryton.Tryton-70
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '45'
+command: tryton
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=home
```

Comment:
Why do you need full home? Does it support filechoosers?

Reply:
Indeed https://foss.heptapod.net/tryton/tryton/-/issues/11941 was integrated in this release.